src/working-mode

```diff
@@ -0,0 +1,252 @@
+<!doctype html>
+<html lang="en">
+ <head>
+  <title>Web Hypertext Application Technology Working Group Working Mode</title>
+  <link rel="stylesheet" href="/style/tabbed-pages">
+  <link rel="icon" href="https://resources.whatwg.org/logo.svg">
+ </head>
+ <body>
+  <h1>
+   <span class="the">The</span>
+   <strong class="what">Web Hypertext Application Technology</strong>
+   <span class="wg">Working Group</span>
+  </h1>
+  <ul class="navigation">
+   <li><a href="/" rel="home">Home</a></li>
+   <li><a href="https://spec.whatwg.org/">Standards</a></li>
+   <li class="this"><strong>Working Mode</strong></li>
+  </ul>
+
+  <h2>Working Mode</h2>
+
+  <p>The WHATWG develops standards and assists in maintaining their corresponding tests. The WHATWG
```

Comment:
The repetition here is a bit unfortunate for the opening paragraph. Will try to massage it a bit.